README.md

```diff
@@ -74,153 +74,146 @@ generally a good idea to pursue feedback from other project developers
 ## The RFC life-cycle
+
 [The RFC life-cycle]: #the-rfc-life-cycle
 
 Once an RFC becomes "active" then authors may implement it and submit the
-change as a pull request to the corresponding Fabric repo. Being "active" is not a rubber
-stamp, and in particular still does not mean the change will ultimately be
-merged; it does mean that in principle all the major stakeholders have agreed
-to the change and are amenable to merging it.
+change as a pull request to the corresponding Fabric repo. Being "active" is
+not a rubber stamp, and it does not mean the change will ultimately be merged;
+it does mean that in principle all the major stakeholders have agreed to the
+change, and are amenable to merging it.
 
 Furthermore, the fact that a given RFC has been accepted and is "active"
 implies nothing about what priority is assigned to its implementation, nor does
-it imply anything about whether a Fabric developer has been assigned the task of
-implementing the feature. While it is not *necessary* that the author of the
+it imply anything about whether a Fabric developer has been assigned the task
+of implementing the feature. While it is not *necessary* that the author of the
 RFC also write the implementation, it is by far the most effective way to see
 an RFC through to completion: authors should not expect that other project
 developers will take on responsibility for implementing their accepted feature.
 
-Modifications to "active" RFCs can be done in follow-up pull requests. We
-strive to write each RFC in a manner that it will reflect the final design of
-the feature; but the nature of the process means that we cannot expect every
-merged RFC to actually reflect what the end result will be at the time of the
-next major release.
+Modifications to active RFCs can be done in follow-up pull requests. We strive
+to write each RFC in a manner that it will reflect the final design of the
+feature; but the nature of the process means that we cannot expect every merged
+RFC to actually reflect what the end result will be at the time of the next
+major release.
 
 In general, once accepted, RFCs should not be substantially changed. Only very
 minor changes should be submitted as amendments. More substantial changes
 should be new RFCs, with a note added to the original RFC. Exactly what counts
 as a "very minor change" is up to the maintainers to decide.
 
 ## Reviewing RFCs
+
 [Reviewing RFCs]: #reviewing-rfcs
 
-While the RFC pull request is up, the maintainers may schedule meetings with the
-author and/or relevant stakeholders to discuss the issues in greater detail,
-and in some cases the topic may be discussed at a sub-team meeting. In either
-case a summary from the meeting will be posted back to the RFC pull request.
+While the RFC pull request is up, the maintainers may schedule meetings with
+the author and/or relevant stakeholders to discuss the issues in greater
+detail, and in some cases the topic may be discussed at a contributors meeting.
```

Comment:
Replaced "sub-team meeting" with "contributors meeting" (all other references to "sub-team" have been changed to "maintainers").